test/is_palindrome.hpp

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+
+bool isPalindrome(int x) {
+    if (x<0) return false;
+    std::string Palindrome = std::to_string(x);
```

Comment:
https://google.github.io/styleguide/cppguide.html#Variable_Names

```suggestion
    std::string palindrome = std::to_string(x);
```